pkgs/development/python-modules/aionotify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aionotify";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01r5y76j889r1h83vgqgzzspjfr5m4q2gxpr6d2h65di1snh5dv4";
+  };
+
+  disabled = pythonOlder "3.5";
+
+  checkInputs = with python3Packages; [
+    asynctest
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/rbarrois/i3paionotify";
+    description = "Simple, asyncio-based inotify library for Python";
+    license = with lib.licenses; [ bsd2 ];
+    platforms = platforms.linux;
```

Comment:
Using the same recommendation as upstream: https://github.com/rbarrois/aionotify/blob/master/setup.py#L49

Makes sense since inotify should be a Linux-only syscall.